event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinMapper.java

```diff
@@ -39,7 +45,29 @@ public final class CristinMapper {
     public static final String PUBLICATION = "publication";
     public static final String RESOURCES = "resources";
     public static final String PERSON = "person";
+    public static final String PARENT_PUBLICATION_SERIES_LEVEL_JSON_POINTER = "/publicationContext/entityDescription"
+                                                                              + "/reference/publicationContext/series"
+                                                                              + "/level";
+    public static final String PARENT_PUBLICATION_SERIES_SCIENTIFIC_VALUE_JSON_POINTER =
```

Comment:
Use constants from `nvi-commons` -> `utils` -> `JsonPointers`?

Reply:
There are other constants, have checked. I do not have entityDescription in "NviReport"